.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can you remove this file from the commit please. rrmrf and then add and commit the changes